### PR TITLE
Get metrics of pull requests from GitHub

### PR DESCRIPTION
**Pull request by jn2657**
# Description

Add functions of getting metrics of pull requests from GitHub

### CODE REVIEW

File: src/main/java/pvs/app/service/GithubApiService.java [hunk 1]
```diff
@@ -232,6 +254,53 @@ public boolean getCommitsFromGithub(String owner, String name, Date lastUpdate)
...
+        Optional<JsonNode> paginationInfo = Optional.ofNullable(mapper.readTree(responseJson))
+                .map(resp -> resp.get("data"))
+                .map(data -> data.get("repository"))
+                .map(repo -> repo.get("pullRequests"));
+
+        if (paginationInfo.isPresent()) {
+            int totalCount = paginationInfo.get().get("totalCount").asInt();
+            List<GithubPullRequestLoaderThread> githubPullRequestLoaderThreadList = new ArrayList<>();
+
+            if (0 != totalCount) {
```
Comment (Xanonymous-GitHub) [inline src/main/java/pvs/app/service/GithubApiService.java:279]:
```suggestion
            if (totalCount > 0) {
```

File: src/main/java/pvs/app/service/GithubApiService.java [hunk 2]
```diff
@@ -74,6 +76,26 @@ private void setGraphQlGetIssuesTotalCountQuery(String owner, String name) {
         this.graphQlQuery = graphQl;
     }
 
+    private void setGraphQlGetPullRequestQuery(String owner, String name) {
+        Map<String, Object> graphQl = new HashMap<>();
+        graphQl.put("query", "{repository(owner: \"" + owner + "\", name:\"" + name + "\") {" +
+                "pullRequests (first: 100) {" +
```
Comment (Xanonymous-GitHub) [inline src/main/java/pvs/app/service/GithubApiService.java:82]: Suggest extracting this number(`100`) to an isolated variable.

Comment (jn2657) [reply src/main/java/pvs/app/service/GithubApiService.java:82]: Include all the other functions in `GithubApiService`?

**Review by hellenGUAN — APPROVED** [review]

**Review by Xanonymous-GitHub — APPROVED** [review]
LGTM

**Pull request merged**